### DISCUSSION TITLE
How to include .trivyignore file when scanning docker images?

### DISCUSSION THREAD

**Discussion by rameshbunnings** [aquasecurity/trivy#1749] [Q&A]
Hello,

I am running docker image scan via a docker run --rm...  from a azure devops pipeline. I can't find a way to pass .trivyignore file while scanning docker image using docker run.

Tried by including the .trivyignore file in the image I wanted to scan, but no luck. Any ideas?

Thanks,
Ramesh

**Comment by fabiolune** [comment]
Maybe it's a bit late, but I had to solve the same issue today.
You just need to mount the `.trivyignore` file inside the container and use the directory where you place it as `workdir` for the running container.
Something like:

``` shell
docker run \
  --rm \  
  -v /var/run/docker.sock:/var/run/docker.sock \
  -v $(pwd)/.trivyignore:/work/.trivyignore -w /work \
  aquasec/trivy image \
  <your image>
```
Hope this still helps,
F
